lib/views/note/note.dart

```diff
@@ -15,33 +15,19 @@ class _NoteWebViewContainerState extends State<NoteWebViewContainer> {
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <style>
```

Comment:
There is trial.html file that has well formated HTML content with style. you can use that for the time being.